src/containers/ListItemContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ListItem from "../components/ListItem";
+
+const ListItemContainer = (props) => {
+  return (
+    <section className="itemSection">
+      <h4 className="sectionTitle">
+        {props.title}(<p className="itemCount">{props.listCnt}</p>)
+      </h4>
+      <div className="itemList invisibleScrollbar">
+        {/* props.listType은 done/todo로 구분된다 */}
+        {props.listState.map((item) => {
+          if (item.type === props.listType) {
+            return (
+              <ListItem
+                key={item.id}
+                item={item}
+                listType={props.listType}
+                modType={props.modType}
+                dispatchListState={props.dispatchListState}
+              />
+            );
+          }
+          return;
+        })}
+      </div>
+    </section>
+  );
+};
```

Comment:
이 부분도 `props`를 destructuring 해 주시면 보다 간결하고 명확하게 코드를 쓸 수 있겠죠?

```suggestion
const ListItemContainer = ({
  title,
  listCnt,
  listState,
  listType,
  modType,
  dispatchListState,
}) => {
  return (
    <section className="itemSection">
      <h4 className="sectionTitle">
        {title}(<p className="itemCount">{listCnt}</p>)
      </h4>
      <div className="itemList invisibleScrollbar">
        {/* props.listType은 done/todo로 구분된다 */}
        {listState.map((item) => {
          if (item.type === listType) {
            return (
              <ListItem
                key={item.id}
                item={item}
                listType={listType}
                modType={modType}
                dispatchListState={dispatchListState}
              />
            );
          }
          return;
        })}
      </div>
    </section>
  );
};
```

Reply:
맞습니다, 안그래도 이런 문법을 찾고 있었는데 기억이 안났었어요.
잘 짚어주셔서 감사합니다~